poller/poller.go

```diff
@@ -107,11 +106,6 @@ func (poller *Poller) pollForChanges() {
 
 }
 
-// GetSplitData returns split data cache results
```

Comment:
Why do we need to get rid of this? I was planning on using it for the change I am working on.

Reply:
Its unused and not a documented part of the package. I could just make it an internal function?

Reply:
Sure, that works!

Reply:
Actually, can you add it back in as internal as part of your PR? I think that makes more sense